renderer/routes/recent.svelte

```diff
@@ -0,0 +1,110 @@
+<script>
+  import { onMount } from 'svelte'
+  import { fade } from 'svelte/transition'
+  import { _ } from 'svelte-intl'
+  import { replace } from 'svelte-spa-router'
+  import { distinct, filter, map } from 'rxjs/operators'
+  import { Heading, Button, PlaylistTracksTable } from '../components'
+  import { load, changes, removals } from '../stores/playlists'
+  import { add } from '../stores/track-queue'
+  import {
+    invoke,
+    formatTimeLong,
+    fromServerChannel,
+    sumDurations
+  } from '../utils'
+
+  let playlist
+
+  $: playlistId = 151120
```

Comment:
in Svelte, `$:` denotes a computed variable. In this case, a simple `const` is enough.
Even better, with `tracks.getRecent()`, we don't need to store it anyway.